__tests__/__packages__/cli-test-utils/src/TestUtils.ts

```diff
@@ -12,7 +12,64 @@
 import * as fs from "fs";
 import { spawnSync, SpawnSyncReturns, ExecFileException } from "child_process";
 import { ITestEnvironment } from "./environment/doc/response/ITestEnvironment";
-import { CommandProfiles, ICommandDefinition, IHandlerParameters } from "@zowe/imperative";
+import { AbstractSession, CommandProfiles, ICommandDefinition, IHandlerParameters, IO } from "@zowe/imperative";
+import { DeleteJobs, ICommonJobParms, IDeleteJobParms, IJob } from "@zowe/zos-jobs-for-zowe-sdk";
+import { Delete } from "@zowe/zos-files-for-zowe-sdk";
+import { posix } from "path";
+
+/**
+ * Delete a local testing file after use
+ * @param {string} filePath - File path of temporary file
+ */
+export function deleteLocalFile(filePath: string): void {
```

Comment:
thanks for exporting these APIs.
That will make it more convenient for plug-in developers using this (`@zowe/cli-test-utils`) package to clean things up after running system tests 😋